.gitignore

```diff
@@ -2,6 +2,8 @@ node_modules
 
 /build
 /public/build
+/public/posts
+/public/blog-post-cache.json
```

Comment:
Just a sanity check, what's the reason for gitignoring these lines? Also same with `/app/_posts`?

Reply:
`/app/_posts` is a holdover from when I was trying a different approach and it looks like I forgot to clean it up. My bad. It's gone now.

As for ignoring `/public/posts` and /public/blog-post-cache.json`, since the blog auto-generates these files on build, it seemed like good practice to ignore these because they could change all the time on everyone's local system, and because everything under `/public/posts` would just be a duplicate of what is under `/blog/posts`. But we can track these files if you think they are important.